Source/PeleLM.cpp

```diff
@@ -6700,8 +6700,8 @@ PeleLM::compute_scalar_advection_fluxes_and_divergence (const MultiFab& Force,
     {
       for (int d=0; d<AMREX_SPACEDIM; ++d)
       {
-        const Box& bx = S_mfi.growntilebox();
-        const Box& ebox = amrex::surroundingNodes(bx,d);
+        const Box& bx = S_mfi.tilebox();
+        const Box& ebox = amrex::surroundingNodes(amrex::grow(bx,2),d);
 
```

Comment:
Why isn't this grown nodal tile box?

Reply:
Maybe the loops can be inverted.  The validbox would be the right thing to tile over naturally?

Reply:
Now I think this should be mfi.nodaltilebox(d), no?